### PR TITLE
Move to webpack

### PR DESCRIPTION
**Pull request by jennie-quinton**

### CODE REVIEW

File: client/package.json [hunk 1]
```diff
@@ -20,11 +18,23 @@
...
+    "mini-css-extract-plugin": "^0.5.0",
     "node-sass": "^4.11.0",
     "react": "^16.8.3",
     "react-dom": "^16.8.3",
-    "redux": "^4.0.1"
+    "redux": "^4.0.1",
+    "sass-loader": "^7.1.0",
+    "style-loader": "^0.23.1",
+    "webpack": "^4.29.5",
+    "webpack-dev-server": "^3.2.1"
```
Comment (jordanrios94) [inline client/package.json]: Move this to `devDependencies` please. The reason is due to production server does not need to install this, therefore making the install command slightly quicker.

File: client/webpack.config.js [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+      template: './index.html',
+      filename: 'index.html',
+    }),
+    new MiniCssExtractPlugin({
+      filename:'index.css'
+    }),
+    new webpack.DefinePlugin({
+      'process.env': {}
+    }),
+    // new webpack.optimize.CommonsChunkPlugin({
```
Comment (jordanrios94) [inline client/webpack.config.js:76]: Is this commented code needed?

**Comment by jordanrios94** [comment]
Brilliant! The client webpack config works for serving the site statically.

When resolving the conflicts, just override them with your changes.

Once merged, I will update my PR to add your new node scripts to be part of the deployment.

File: client/webpack.config.js (hunk 2 above)
Comment (jennie-quinton) [reply client/webpack.config.js:76]: not yet, but will do if we have bundles (ie multiple pages), so I left it there so I can uncomment when needed, i can remove though?

Comment (jordanrios94) [reply client/webpack.config.js:76]: no need to remove, just wanted to know why it was left

**Pull request merged**